resources/env/authoring/bin/crafter-setenv.sh

```diff
@@ -141,6 +141,22 @@ export CRAFTER_SYSTEM_ENCRYPTION_SALT=${CRAFTER_SYSTEM_ENCRYPTION_SALT:="DgGN9xh
 # -------------------- Configuration variables --------------------
 export CRAFTER_ENVIRONMENT=${CRAFTER_ENVIRONMENT:=default}
 
+# -------------------- Serverless variables --------------------
+export AWS_S3_ENDPOINT=${AWS_S3_ENDPOINT:=""}
+export AWS_S3_PATH_STYLE_ACCESS=${AWS_S3_PATH_STYLE_ACCESS:="false"}
+
+export SERVERLESS_NAMESPACE=${SERVERLESS_NAMESPACE:=cloud-sites}
+
+export PREVIEW_BUCKET_NAME_PATTERN=${SERVERLESS_NAMESPACE}-blobs-\${siteName}
+export PREVIEW_BUCKET_PREFIX_PATTERN=sandbox
```

Comment:
I don't thing we need to add a prefix for preview. Since it's a different bucket than the others, normally this is empty

Reply:
Notice these variables can be used for non-serverless as well. We could configure blob-stores [like this ](https://docs.craftercms.org/en/4.1/reference/modules/studio/configuration/blob-stores.html#sample) and share the S3 buckets between sites

Reply:
Right but I thought we discussed that we'll try to have this config as close as possible to Cloud

Reply:
I see. So let's keep the config but default to empty

Reply:
Ok. I'll approve the PR as soon as you leave the variable empty